partner-portal/src/API/Model/DocumentFactory.cs

```diff
@@ -0,0 +1,49 @@
+﻿using EnumsNET;
+using Google.Protobuf.WellKnownTypes;
+using Pssg.SharedUtils;
+using Rsbc.Dmf.CaseManagement.Service;
+
+namespace Rsbc.Dmf.PartnerPortal.Api.Model
+{
+    public class DocumentFactory
+    {
+        public LegacyDocument Create(Driver driver, string userId, string documentUrl, string documentType, string documentTypeCode, string caseId = "")
```

Comment:
this is my domain cross-cutting concern bug, hopefully I have time to clean this up in all the portals. Ideally, I'd move this logic to cms-adapter